enterprise/api/v1/views/enterprise_group.py

```diff
@@ -112,16 +113,19 @@ def get_learners(self, *args, **kwargs):
                         'learner_uuid': 'enterprise_customer_user_id',
                         'pending_learner_id': 'pending_enterprise_customer_user_id',
                         'enterprise_group_membership_uuid': 'enterprise_group_membership_uuid',
+                        'enterprise_customer': EnterpriseCustomerSerializer
                     },
                 ],
             }
 
         """
 
         group_uuid = kwargs.get('group_uuid')
+        # GET api/v1/enterprise-group/<group_uuid>/learners?filter_for_pecus=true
+        filter_for_pecus = self.request.query_params.get('filter_for_pecus', None)
```

Comment:
You can change this method signature to have a `request` param, so instead of `def get_learners(self, *args, **kwargs)`, make it `def get_learners(self, request, *args, **kwargs)`.  This you can do `request.query_params.get(...)` on this line.